codec-server/codec-server/main.go

```diff
@@ -138,6 +138,6 @@ func main() {
 	case <-sigCh:
 		_ = srv.Close()
 	case err := <-errCh:
-		logger.Fatal("error", tag.NewErrorTag(err))
+		logger.Fatal("error", tag.NewErrorTag("error", err))
```

Comment:
```suggestion
		if err != http.ErrServerClosed {
				logger.Fatal("error from HTTP server", tag.NewErrorTag(err))
		}
```

Reply:
I guess technically we should wait for `ListenAndServe` to complete too but that's out of scope for this PR.